assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -63,29 +71,20 @@
 
   @include small-and-less {
     width: 100vw;
-    inset-inline-start: -100vw;
-
-    @supports not (inset-inline-start: -100vw) {
-      transform: translateX(-100vw);
-      transition: transform 0.2s ease-in;
-
-      html[dir="rtl"] & {
-        transform: translateX(100vw);
-      }
-    }
   }
 
-  @include small-to-large {
```

Comment:
Yes that was because I removed both the `100vw` and `400px` variant of the code, whereas only the `100vw` had the same effect as `100%`. I re-added it in the latest push, can you confirm it's ok now?